assignment/api/src/resources/assignment_ca.properties

```diff
@@ -1124,9 +1127,19 @@ associate.label=Usa la r\u00fabrica seg\u00fcent per a qualificar l\u2019activit
 option.pointsoverride=Ajusta les puntuacions individuals dels estudiants
 option.hidepoints=Amaga les puntuacions (nom\u00e9s comentaris)
 option.studentpreview=Amaga la r\u00fabrica a  l\u2019estudiant
+option.selfreport=Permetre que els estudiants s'autoavalu\u00EFn la seva tasca
```

Comment:
Please, put all the new translations at the end of the file with the S2U-34 prefix